src/CONST.ts

```diff
@@ -803,7 +803,7 @@ const CONST = {
     },
 
     // at least 8 characters, 1 capital letter, 1 lowercase number, 1 number
-    PASSWORD_COMPLEXITY_REGEX_STRING: '^(?=.*[A-Z])(?=.*[0-9])(?=.*[a-z]).{8,}$',
+    PASSWORD_COMPLEXITY__STRING: '^(?=.*[A-Z])(?=.*[0-9])(?=.*[a-z]).{8,}$',
```

Comment:
I wanted to ask why this regex has been renamed, but it seems it is not used anywhere in the codebase. Could we remove it in this PR @arosiclair?

Reply:
Yeah please revert this change @Tony-MK it doesn't seem necessary

Reply:
Sorry for that; it is a mistake. Let me revert it.

Reply:
@arosiclair, I mean this regex is not used anywhere. Can we remove it in this PR to make the code cleaner?